webpack/assets/javascripts/react_app/redux/actions/statistics.js

```diff
@@ -0,0 +1,16 @@
+import API from "../../API";
+import {
+  STATISTICS_DATA_REQUEST,
+  STATISTICS_DATA_SUCCESS,
+  STATISTICS_DATA_FAILURE
+} from "../consts";
+
+export const getStatisticsData = charts => dispatch => {
+  dispatch({ type: STATISTICS_DATA_REQUEST, payload: charts });
+  charts.map(c => c.url).forEach(url => {
+    API.get(url).then(
+      result => dispatch({ type: STATISTICS_DATA_SUCCESS, payload: result }),
+      (jqXHR, textStatus, errorThrown) => dispatch({ type: STATISTICS_DATA_FAILURE, payload: errorThrown })
+    );
+  });
+};
```

Comment:
Am I missing something or isn't this file redundant with 'redux/actions/statistics/index.js' below?